website/templates/search.mako

```diff
@@ -119,7 +119,13 @@
     </div>
 
     <script type="text/html" id="file">
-        <h4><a data-bind="attr: {href: guid_url || deep_url}, text: name"></a> (<span class="text-danger" data-bind="if: is_retracted">Withdrawn </span><span data-bind="if: is_registration">Registration </span>File)</h4>
+        <!-- ko if: guid_url || deep_url -->
+            <h4><a data-bind="attr: {href: guid_url || deep_url}, text: name"></a> (<span class="text-danger" data-bind="if: is_retracted">Withdrawn </span><span data-bind="if: is_registration">Registration </span>File)</h4>
+        <!-- /ko-->
+        <!-- ko ifnot: guid_url || deep_url -->
+            <h4> <span data-bind="text:name"></span> (Preprint File)</h4>
```

Comment:
Preprint files in search don't have a link since we are blocking the file detail page for preprints.